static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -528,44 +502,9 @@ export default class AudioPlayer extends Component<Props, State> {
                     {this.state.ready ? <Time t={this.state.duration} /> : ''}
                 </TimeContainer>
                 <WaveAndTrack>
-                    <Wave
-                        innerRef={this.setCanvas}
-                        colour={pillarsHighlight[`${this.props.pillar}`]}>
-                        {this.state.ready
-                            ? this.state.bins.map(
-                                  (barHeight, i) =>
-                                      i < this.state.iteration ? (
-                                          <rect
-                                              x={i * (this.props.barWidth + 1)}
-                                              y={this.state.canvasH - barHeight}
-                                              width={this.props.barWidth}
-                                              height={barHeight}
-                                              fill={
-                                                  pillarsHighlight[
-                                                      `${this.props.pillar}`
-                                                  ]
-                                              }
-                                          />
-                                      ) : i < this.state.iteration + 1 ? (
-                                          <rect
-                                              x={i * (this.props.barWidth + 1)}
-                                              y={this.state.canvasH}
-                                              width={this.props.barWidth}
-                                              height={0}
-                                              fill={palette.neutral[4]}
-                                          />
-                                      ) : (
-                                          <rect
-                                              x={i * (this.props.barWidth + 1)}
-                                              y={this.state.canvasH - barHeight}
-                                              width={this.props.barWidth}
-                                              height={barHeight}
-                                              fill={palette.neutral[4]}
-                                          />
-                                      )
-                              )
-                            : ''}
-                    </Wave>
+                    <FakeWave>
+                        <span ref={this.setWave} className="wave-holder" dangerouslySetInnerHTML={{ __html: waveW.markup }}></span>
```

Comment:
Switch to `DIV` here, that may explain why you have to `transform` the `svg` as spans are inline elements and are position on line boxes according to the baseline.